core/modules/script.py

```diff
@@ -45,18 +82,33 @@ class ScriptNodeSkel(BaseScriptAbstractSkel):
         defaultValue=False,
     )
 
-    plugin = BooleanBone(
-        descr="Is plugin?",
-        defaultValue=False
-    )
-
     name = StringBone(
         descr="Folder",
         required=True,
         vfunc=lambda value: not DIRECTORY_PATTERN.match(value)
     )
 
 
+class ButtonbarSkel(skeleton.RelSkel):
+    module = SelectBone(
+        descr="Modul",
+        values=_get_modules_or_handlers
+    )
+    capable = SelectBone(
+        descr="Capable to handle",
+        values={
+            "no_keys": "Running without selecting anything",
+            "one_key": "Script may use one database key in params",
+            "several_keys": "Script may use several database keys in params"
+        }
+    )
+    thismoduleaccess = SelectBone(
+        descr="Required access rights of specific modules",
+        values=["view", "add", "edit", "delete"],
+        multiple=True,
+    )
```

Comment:
What's the purpose of this bone?

Reply:
You may start the script in context of one module. Since we dont know which module is selected in advance i chose to provide the accessvalues without modulenames.